protos/sigstore_bundle.proto

```diff
@@ -44,13 +44,13 @@ message TimestampVerificationData {
         repeated dev.sigstore.common.v1.RFC3161SignedTimestamp rfc3161_timestamps = 1;
 }
 
-// VerificationData contains extra data that can be used to verify things
+// ExtendedVerificationMaterials contains extra data that can be used to verify things
```

Comment:
I would prefer to not call this Extended because it suggests these are optional. They’re not - you need a timestamping element, and you need transparency. You _can_ use Sigstore without those, but that’s not the value ads for Sigstore.

Reply:
100% agree with your reasoning. Thoughts on directly decomposing it into the `Bundle` body instead? That was my other idea in #30.

Reply:
To clarify, you're saying to remove this intermediate message and just have these two fields be a part of `Bundle`?

Reply:
> To clarify, you're saying to remove this intermediate message and just have these two fields be a part of Bundle?

Yep, that's what I was thinking!

Reply:
SG to me!